docs/how-to/encryption.md

```diff
@@ -0,0 +1,125 @@
+# Managing Encryption
+
+The Apache Cassandra charm has encryption enabled by default.
+All units within a cluster share the same CA certificate file, but each unit has a distinct private key.
+
+This charm implements the **Requirer** side of the [`tls-certificates/v1`](https://github.com/canonical/charm-relation-interfaces/blob/main/interfaces/tls_certificates/v1/README.md) relation. Therefore, any charm implementing the **Provider** side can be used.
```

Comment:
Suggestion: Avoid using the inline code blocks syntax in links text and consider replacing the link with this one - https://charmhub.io/integrations/tls-certificates

```suggestion
This charm implements the **Requirer** side of the [tls-certificates](https://charmhub.io/integrations/tls-certificates) relation. Therefore, any charm implementing the **Provider** side can be used.
```